ecommerce/views.py

```diff
@@ -24,6 +23,7 @@ def save_order_to_session(session, order: Order):
     session.modified = True
 
 
+# TODO - rm it
```

Comment:
Let's create a properly formatted todo for this

Reply:
just removed it